### PR TITLE
Fix CSS display rule

### PR DESCRIPTION
**Pull request by JosefJezek**
This fix resolve display in IE10...

### CODE REVIEW

File: marked-element.html [hunk 1]
```diff
@@ -66,6 +66,8 @@
 <dom-module id="marked-element">
   <template>
     <style>
+      display: inline-block;
```
Comment (notwaldorf) [inline marked-element.html]: This needs to be in a `:host` probably

**Comment by JosefJezek** [comment]
@notwaldorf I added `:host`

**Comment by notwaldorf** [comment]
Thanks!

**Comment by bashmish** [comment]
@JosefJezek can you please explain what issue in IE10 you wanted to solve changing default display property?
Also I don't understand why this pull request was merged without any explanation of what exactly was going to be fixed.

**Comment by notwaldorf** [comment]
I think I merged it because custom elements are `display:inline` by default, which has caused IE sizing problems in the past. I think that we should change this to `display:block`, however.

**Pull request merged**